### PR TITLE
Fixes issue #115 - issues with "edge mask" aka aperture reduction

### PR DESCRIPTION
**Pull request by gr5**
When reading old OLN files, ignore edge mask
When writing OLN files don't save edge mask
Fixed 2 locations where edge mask needs to be set to zero when checkbox is not checked in mirror dialog Fixed bug where edge mask is checked if not found in OLN file!

### CODE REVIEW

File: igramarea.cpp [hunk 1]
```diff
@@ -2140,45 +2142,6 @@ void IgramArea::loadOutlineFile(QString fileName){
...
-                break;
-            case QMessageBox::No:
-            default: // if they click red X do same thing as "no" button
-                md.changeEdgeMaskvalues(md.aperatureReduction);
-                break;
-            }
-        }
-    }
-    else{ // just enable edge mask check box to use the current value.
-        md.changeEdgeMaskvalues(md.aperatureReduction);
```
Comment (gr5) [inline igramarea.cpp:2180]: So this line of code is particularly nasty and found when reading OLN file: both the older V6 style OLN file and also when reading V7.  It checks the "edge mask" checkbox even if nothing is found regarding edge mask in the OLN file!

**Comment by gr5** [comment]
I spent a lot of time testing all the changes.  It seems to work fine now.

**Review by githubdoe — APPROVED** [review]
Thanks.  Looks like it was pretty messed up with a lot of omissions.  Makes me wonder what I was thinking.  Guess I was distracted.  Did not realize all the implications.  Still puzzled why I did so much work in the reading and writing of it in the .oln file.  That worries me that I thought it necessary.  Now of course I think it was not.

**Review by githubdoe — APPROVED** [review]

**Comment by gr5** [comment]
Dale, Do you want me to release 7.2 now just for this change?  It's not too much trouble.  I need to test the final release if I do a release it but I'll probably just do a few quick tests (load igram, process) as not much changed.  And I already tested the changes.  Although I will need to update the revision history and you'll need to approve that.

**Comment by githubdoe** [comment]
I think it would be nice but I will leave it up to you to decide based on your schedule and such.

File: igramarea.cpp [hunk 2]
```diff
@@ -2257,40 +2220,7 @@ void IgramArea::loadOutlineFileOldV6(QString fileName){
         mirrorDlg &md = *mirrorDlg::get_Instance();
```
Comment (atsju) [inline igramarea.cpp:2220]: This line can be deleted as `md` is not used anymore. This will fix the `[-Wunused-variable]`

**Comment by atsju** [comment]
> Dale, Do you want me to release 7.2 now just for this change? It's not too much trouble. I need to test the final release if I do a release it but I'll probably just do a few quick tests (load igram, process) as not much changed. And I already tested the changes. Although I will need to update the revision history and you'll need to approve that.

I wonder if this should be v7.1.2. This would be a pure bugfix release. 
Sorry if this seems futile and great work to find and fix this. Thank you

File: igramarea.cpp (hunk 2 above)
Comment (gr5) [reply igramarea.cpp:2220]: oops.  Already merged.  Didn't see your comment until now.

Comment (atsju) [reply igramarea.cpp:2220]: you can fix in the other PR. It's really minor. 
This is how dead code comes to live :ghost:

**Pull request merged**